experimental/packages/otlp-transformer/src/trace/types.ts

```diff
@@ -14,14 +14,9 @@
  * limitations under the License.
  */
 
-import { Fixed64, IInstrumentationScope, IKeyValue } from '../common/types';
-import { IResource } from '../resource/types';
 
-/** Properties of an ExportTraceServiceRequest. */
-export interface IExportTraceServiceRequest {
-  /** ExportTraceServiceRequest resourceSpans */
-  resourceSpans?: IResourceSpans[];
-}
+import { TFixed64, TInstrumentationScope, TKeyValue, TResource } from '../common/types';
+import { Type, type Static } from "@sinclair/typebox";
 
 export interface IExportTraceServiceResponse {
```

Comment:
Note that I left these top-level "response" types untouches (so this PR doesn't actually change the deserialization behavior - it just does the ground work required to make it work)

The reason why is because right now the deserialization code is just some placeholders and I wasn't sure what you had planned for how to use them. Instead of making an opinionated changed, I wanted to keep the scope of this PR small where it just introduces the groundwork required for deserialization, then updating the deserialization logic can be done in a separate PR